docs/contributing/playbook.md

```diff
@@ -0,0 +1,104 @@
+|                               **Interface**                               |                                                           :material-check: [MMC interface](../assets/images/hyperv-manager.png), no tabs                                                           |                                                                     :material-check-all: [Dark mode, tabbed, easier](../assets/images/vmware-workstation.png)                                                                      |
+|                             **Snapshotting**                              |                                                     :material-check-all: Very fast by default, worse UI as it's a list instead of a grid view                                                      |     :material-check: Better UI, but you should [disable taking snapshots in background](https://docs.vmware.com/en/VMware-Workstation-Pro/17/com.vmware.ws.using.doc/GUID-AB7628AA-16CD-4380-AF52-C1716A1EEE10.html) for speed     |
+|                         **Ease of Installation**                          |          :material-check: Installation through [Windows features](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v), no external downloads           | :material-alert-circle-outline: It's [officially free now](https://blogs.vmware.com/workstation/2024/05/vmware-workstation-pro-now-available-free-for-personal-use.html), but you need to tediously sign-up for a Broadcom account |
+|                             **Compatibility**                             |                                                     :material-alert-circle-outline: Enables 'Secure system,' which causes issues for some apps                                                     |                                                                       :material-check-all: No incompatibilities with apps as it's not directly in the kernel                                                                       |
+|                              **Guest Tools**                              | :material-alert-circle-outline: Annoying to work with as it's [just an RDP session](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/enhanced-session-mode), low FPS |                                                                        :material-check-all: Has a vGPU, 60fps viewport, good drag-and-drop support, stable                                                                         |
+
+!!! note "Apple-silicon Mac users"
+
+    Due to not being tested frequently, we would appreciate it if you could test using Atlas with Windows-on-ARM using your Apple-silicon-based device. For this, consider using either:
```

Comment:
It's best to mention these apps provide the Windows 11 ARM64 installers since we don't offer them ourselves.